### PR TITLE
tp-qemu: host-kernel: Add RHEL.6.7 host kernel config.

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1251372

### CODE REVIEW

File: qemu/cfg/host-kernel/Host_RHEL/6/7.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- 7:
+    host_kernel_ver_str += ".7"
+    requires_kernel = [">= 2.6.32-505", "<  2.6.32-574"]
+    machine_type = "rhel6.6.0"
```
Comment (xutian) [inline qemu/cfg/host-kernel/Host_RHEL/6/7.cfg]: machine_type = "rhel6.7.0" ?

File: qemu/cfg/host-kernel/Host_RHEL/6/7.cfg [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    set_link:
+        operstate_always_up = no
+    cpuinfo_query.qmachine_type:
+        cpu_info = "pc,rhel.6.6.0,rhel6.5.0,rhel6.4.0,rhel6.3.0,rhel6.2.0,rhel6.1.0,rhel6.0.0,rhel5.5.0,rhel5.4.4,rhel5.4.0"
```
Comment (xutian) [inline qemu/cfg/host-kernel/Host_RHEL/6/7.cfg]: rhel6.7.0 ?

File: qemu/cfg/host-kernel/Host_RHEL/6/7.cfg (hunk 1 above)
Comment (spcui) [inline qemu/cfg/host-kernel/Host_RHEL/6/7.cfg]: Hi @xutian 
Haven't "rhel6.7.0" machine type now.

Comment (CongSmile) [inline qemu/cfg/host-kernel/Host_RHEL/6/7.cfg]: Yes, @spcui is correct.

There is no "rhel.6.7.0" machine type.

**Comment by xutian** [comment]
Acked-by: Xu Tian <xutian@redhat.com>

**Comment by spcui** [comment]
Acked-by: scui@redhat.com

**Comment by xutian** [comment]
Merged, thanks @CongSmile

**Pull request merged**